src/test/scala/nl.knaw.dans.easy.umd/IsniTransformerSpec.scala

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.umd
+
+import org.scalatest.{FlatSpec, Inside, Matchers, OptionValues}
+
+import scala.xml.PrettyPrinter
+
+class IsniTransformerSpec extends FlatSpec with Matchers with OptionValues with Inside {
+
+
+  "EMD <orgISNI>" should "add ISNI organizationId" in {
+    val inputXML =
+      <emd:easymetadata>
+        <emd:title>
+          <dct:alternative>BAAC Rapport 0427</dct:alternative>
+        </emd:title>
+        <emd:creator>
+          <eas:creator>
+            <eas:title>drs</eas:title>
+            <eas:initials>A</eas:initials>
+            <eas:surname>B</eas:surname>
+            <eas:organization>BAAC</eas:organization>
+            <eas:entityId eas:scheme="DAI"></eas:entityId>
+          </eas:creator>
+          <eas:creator>
+            <eas:title>drs</eas:title>
+            <eas:initials>MC</eas:initials>
+            <eas:surname>R</eas:surname>
+            <eas:organization>BAAC bv</eas:organization>
+            <eas:entityId eas:scheme="DAI"></eas:entityId>
+          </eas:creator>
+        </emd:creator>
+      </emd:easymetadata>
+
+    val expectedXML =
+      <emd:easymetadata>
+        <emd:title>
+          <dct:alternative>BAAC Rapport 0427</dct:alternative>
+        </emd:title>
+        <emd:creator>
+          <eas:creator>
+            <eas:title>drs</eas:title>
+            <eas:initials>A</eas:initials>
+            <eas:surname>B</eas:surname>
+            <eas:organization>BAAC</eas:organization>
+            <eas:entityId eas:scheme="DAI"></eas:entityId>
+          </eas:creator>
+          <eas:creator>
+            <eas:title>drs</eas:title>
+            <eas:initials>MC</eas:initials>
+            <eas:surname>R</eas:surname>
+            <eas:organization>BAAC bv</eas:organization>
+            <eas:organizationId eas:identification-system="http://isni.org/isni/0000000472370000" eas:scheme="ISNI">ISNI: 0000 0004 7237 0000</eas:organizationId>
```

Comment:
A "what's in a name" question:

What has an id-**system** to do with and id-**instance**? Shouldn't this become something like:
```suggestion
            <eas:organizationId eas:identification-system="http://isni.org/isni/" eas:scheme="ISNI">ISNI: 0000 0004 7237 0000</eas:organizationId>
```

Reply:
you're right of course 👍